docs/src/test-api/class-testproject.md

```diff
@@ -265,20 +265,20 @@ The maximum number of retry attempts given to failed tests. Learn more about [te
 
 Use [`property: TestConfig.retries`] to change this option for all projects.
 
+## property: TestProject.run
```

Comment:
- Should you remove `TestProject.canShard` from this file?
- Let's also mention how this property affects sharding.